keyboards/crkbd/keymaps/nimishgautam/rules.mk

```diff
@@ -0,0 +1,13 @@
+RGB_MATRIX_ENABLE = yes
+OLED_ENABLE     = yes
+OLED_DRIVER     = SSD1306
+COMBO_ENABLE = yes
+TAP_DANCE_ENABLE = yes  # enable tap dance features
+EXTRAKEY_ENABLE = no
+GRAVE_ESC_ENABLE = no
+SPACE_CADET_ENABLE = no
+SPLIT_KEYBOARD = yes
+#WPM_ENABLE = yes
+#CAPS_WORD_ENABLE = yes # if you hold down both shifts, you get lshift enabled for 1 word, but only works with KC shift, not MOD shift :(
```

Comment:
Mod shift?

Reply:
Yeah, when I tested it with my keyboard, it worked fine if you hold down 2 keys that send   `KC_LSFT` and `KC_RSFT` respectfully but it did NOT work when the keys send something like  `MT(MOD_LSFT, ...) `, `MT(MOD_RSFT, ...) `,  So, it seems to work for the `KC` shifts but not the `MOD` shifts. Not sure if there's a better way to word that.

Reply:
Aah, okay, this is due to mod tap using weak mods, and caps words not checking weak mods.  That ... is actually a simple fix. 

https://github.com/qmk/qmk_firmware/blob/245f00b4d93c65d5540d2096c9f3536ead93ef26/quantum/process_keycode/process_caps_word.c#L26-L28

Changing that would allow for weak mods to be checked.  However, it has the caveat of shifted keycodes (eg `LSFT(kc)`) will also be able to trigger it.